inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/table/DorisDynamicSchemaOutputFormat.java

```diff
@@ -503,6 +511,20 @@ private void handleDirtyData(Object dirtyData, DirtyType dirtyType, Exception e)
                 LOG.warn("Dirty sink failed", ex);
             }
         }
+
+        if (multipleSink) {
+            if (DirtyType.DESERIALIZE_ERROR.equals(dirtyType) || !(dirtyData instanceof RowData)) {
+                metricData.outputDirtyMetricsWithEstimate(null, null, null, rowSize, dataSize);
```

Comment:
why not to pass databse table ?

Reply:
bacause there is no database/table avaliable to parse since the dirty data can't be deserialized, so only add global count in this case